config/cscs.py

```diff
@@ -781,6 +805,18 @@
             'cxx': 'g++',
             'ftn': 'gfortran'
         },
+        {
+            'name': 'PrgEnv-pgi-nompi-nocuda',
+            'target_systems': [
+                'arolla'
```

Comment:
```suggestion
                'arolla', 'tsa'
```

And remove the same entry for Tsa below.